ansible/roles/ceph-grafana/tasks/grafana_plugins.yml

```diff
@@ -15,3 +15,6 @@
 - name: Update Grafana plugins
   command: "grafana-cli plugins update {{ item }}"
   with_items: "{{ grafana_plugins }}"
+
+- name: Update status-panel for readability within the 'light' theme
+  command: "sed -i.bak -e 's/green/rgb(1,167,1)/g' /var/lib/grafana/plugins/vonage-status-panel/dist/css/status_panel.css"
```

Comment:
We can fix the shade of green later. Not too concerned about the ansible warning at this time